### ISSUE TITLE
Field sometimes doesn't load fully

### ISSUE THREAD

**Issue by L11R** [sdwolfe32/ovrstat#22]
Example:
![image](https://user-images.githubusercontent.com/5345489/36876102-61bc0642-1dc4-11e8-8989-4a45dc6475d9.png)
```html
{count, plural, one {Game} other {Games}} Lost
Final {count, plural, one {Blow} other {Blows}} - Avg per 10 Min
```
I think we should always pick "other" plural form as default.

**Comment by L11R** [comment]
Fixed in #23

**Comment by sdwolfe32** [comment]
Thanks for the fix!
